PULL_REQUEST_TEMPLATE.md

```diff
@@ -16,14 +16,10 @@
 
 - [ ] I think the code is well written
 - [ ] Unit tests for the changes exist
+- [ ] Unit tests have been run successfully for: _add or delete as necessary_
```

Comment:
This information is present in CI reports, I do not see any added value here.

Reply:
[From its documentation](https://docs.travis-ci.com/user/reference/windows/), Travis CI seems to support only Python 2.7.9 on Windows:

> A basic Python 2.7.9 interpreter is also included: /C/ProgramData/chocolatey/bin/python.exe

Furthermore, it is unclear if we can do anything with that "basic Python interpreter"

`aiosmtpd` is claimed to run on Linux and Windows, and that is I suggest the PR template to contain info on what platforms the tests have been run.